src/main/resources/i18n/Resources_zh-SG.properties

```
@@ -0,0 +1,55 @@
+# suppress inspection "UnusedProperty" for whole file
+googleStorageWarning=所有上传文件皆使用原始质量大小，会占用Google账号容量。
+# this is a tooltip that pops up when Google API quota is exceeded; the app then slows down by applying exponential backoff; the backoff delay in seconds
+# is appended to the phrase below
+backoffNotice=由于Google的限制，这是一个正常的现象。请稍后几分钟，有时几小时后会继续执行。请耐心等候或停止上传稍后再试。目前等待时间，秒:
+failuresDialogTitlePrefix=错误:
+failuresDialogPathColumnTitle=路径
+failuresDialogFailureColumnTitle=错误项目
+progressBoxFailuresHyperlinkText=错误
+menuItemMacPreferences=偏好...
+menuItemDefaultFile=文件
+menuItemDefaultAbout=关于...
+menuItemDefaultSettings=设置...
+menuItemDefaultExit=关闭
+folderSelectorAlreadyUploadedLabelPrefix=已上传:
+folderSelectorDirectoryChooserTitle=选择媒体文件匣
+mainScreenAboutDialogTitle=关于
+mainScreenPreferencesDialogTitle=偏好设置
+uiAuthorisationBrowserTitle=登录Google
+uploadPaneLogAreaSuccessLabel=全部成功上传!
+uploadPaneLogAreaFailurePrefix=有错误产生:
+folderSelectorDragHereLabel=拖曳文件匣或文檔到这里，或
+folderSelectorBrowseButtonLabel=导航...
+folderSelectorResumeCheckboxLabel=从上次继续
+mainScreenMenuActionsText=动作
+mainScreenMenuActionsStopUploadText=停止上传
+mainScreenMenuActionsLogoutText=注销
+mainScreenPreferencesDialogIgnoreSectionLabel=忽略这些文档或是文件匣
+supportPanePaymentSentence=这是我利用空余时间写的软件。
+supportPanePaymentLinkSentence=请支持这个软件!
+supportPaneReportIssueSentence=有疑问? 任何建议?
+supportPaneReportIssueLinkSentence=请在这里提出问题。
+uploadPaneStopButtonText=停止
+uploadPaneUploadMoreButtonText=继续上传...
+directoryStructureSupplierProgressTitle=扫描文件匣
+cloudAlbumsProviderProgressTitle=撷取Google相册中的影集资料
+uploaderDirectoryProgressTitle=文件匣处理中
+uploaderFileProgressTitle=上传媒体文档
+# the %s here will be substituted with the number of albums
+albumManagerProgressStatusTitlePattern=与Google相册同步%s影集中
+# this is the label on top of the failures table; this table is accessible when there are upload failures;
+# each row in the table shows a path to the file and a problem associated with this path
+failuresDialogTopHint=每一条项目代表单一文档错误，并不影响上传程序。请待上传结束后尝试修正每一项目，再上传一次。列表内项目可以复制至剪贴板。
+aboutDialogVersionLabel=版本
+aboutDialogCreditsText=程序设计@ylexus \ 中文翻译axylee
```

```suggestion
aboutDialogCreditsText=程序设计@ylexus \\ 中文翻译axylee
```

backslash \ needs to be escaped in java property files, please replace all backslashes in all files with double backslashes as I suggested above.